env.js

```diff
@@ -18,7 +18,7 @@ const defaults = {
   RECAPTCHA_SITE_KEY: '6LcyeXoUAAAAAFtdHDZfsxncFUkD9NqydqbIFcCK',
   RECAPTCHA_ENABLED: false,
   CLIENT_ANALYTICS_ENABLED: false,
-  ONBOARDING_MODAL: false,
```

Comment:
Do we really want to change the ENV variables? Activating the onboarding modal by default seems out of scope

Reply:
Replied on Slack

Reply:
@znarf had said to @jeffin143 we can go ahead and make it the default and remove the flags https://opencollective.slack.com/archives/C0RMV6F8C/p1589207632341300?thread_ts=1589206808.339800&cid=C0RMV6F8C